src/alerting/reducers/alertBuilder.test.ts

```diff
@@ -276,4 +305,37 @@ describe('alertBuilderReducer', () => {
       expect(actual.thresholds).toEqual([critThresh])
     })
   })
+
+  describe('updateName', () => {
+    it('changes name', () => {
+      const state = mockState()
+
+      const name = 'new-name'
+
+      const expected = {...state, name}
+
+      const actual = alertBuilderReducer(state, updateName(name))
+
+      expect(actual).toEqual(expected)
+    })
+  })
+
+  describe('default action', () => {
+    it('changes name', () => {
+      const state = mockState()
+      const expected = state
+      const actual = alertBuilderReducer(state, '' as any)
+
+      expect(actual).toEqual(expected)
+    })
+  })
+
+  describe('initial state', () => {
+    it('changes name', () => {
+      const expected = alertBuilderReducer(initialState(), '' as any)
```

Comment:
This is mistake made by me. It should be `{type: '' as any}` and it need to be retyped as any because it will throw an error that '' isn't any of constant strings defined as action types for alertBuilderReducer.

Reply:
could we import and use those constants rather than casting to `any`?

Reply:
found out that type can be undefined so no more casting required